### PR TITLE
Consistent type docblock parsing

### LINKED ISSUE #10322: Cannot use multiline for `@template-use`
A newline in `@use` results in InvalidDocblock, while `@extends` supports multiline.

* `@use`: https://psalm.dev/r/ef05bbff9f
* `@extends`: https://psalm.dev/r/7aacc5733e

Thank you.

### PR DESCRIPTION
**Pull request by robchett**
fixes #10322 #10100 #10095 #10089

### CODE REVIEW

**Comment by robchett** [comment]
Also fixes #10100

File: src/Psalm/Internal/Cli/Psalm.php [hunk 1]
```diff
@@ -272,7 +272,7 @@ public static function run(array $argv): void
 
         self::emitMacPcreWarning($options, $threads);
 
-        self::restart($options, $threads, $progress);
+        //self::restart($options, $threads, $progress);
```
Comment (orklah) [inline src/Psalm/Internal/Cli/Psalm.php:275]: that should stay :)

Comment (robchett) [reply src/Psalm/Internal/Cli/Psalm.php]: Yes it should, forgot to exclude that (it's how I get psalm to run with Xdebug, do let me know if there is a better way). Now undone

**Comment by orklah** [comment]
Thanks!

**Pull request merged**